source/storybook.md

```diff
@@ -24,6 +24,8 @@ This feature is still experimental. We'd be glad to hear about your feedback on
 - The default config will look for your Vulcan local installation based on environment variable. 
   - **If your install folder is named exactly "Vulcan"**, setup your `METEOR_PACKAGE_DIRS`  as usual. Vulcan Loader will automatically detect your install.
   - **If your Vulcan install folder have a specific name**, setup a `VULCAN_DIR` environment variable to point to the **root** path of your Vulcan install.
+- You should be running on node v8.16.0
+  - To install node, run `nvm install 8.16.0` in your terminal
```

Comment:
Does it work with older versions of Node ? Or is it at least v8.16.0?

Reply:
I had node v8.1.0 installed and it complained that the spread operator (used in webpack.config.js) is not supported in that version:
```
module.exports = ({ config }) => {
  // Define aliases. Allow to mock some packages.
  config.resolve = {
    ...config.resolve,
    // this way node_modules are always those of current project and not of Vulcan
    alias: {
      ...config.resolve.alias,
      . . .
    },
  };
  . . .
}
```
Then I upgraded to node v9.11.2 but then another package (style-loader, css-loader, or sass-loader - I forget) complained that it does not work with node versions past v8.x.x. So I settled on the last version 8, which is v8.1.0.